.gitignore

```diff
@@ -1,2 +1,40 @@
+.idea/
+
+# Compiled class file
+*.class
+
+# Log file
+*.log
+
+# BlueJ files
+*.ctxt
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
+*.nar
+*.ear
+*.zip
+*.tar.gz
+*.rar
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+
 .gradle
```

Comment:
make this  `.gradle/`  because it has committed the gradle folder

Reply:
You need to remove it from git and make sure it is ignored.
Always do a `git status -s` before  staging files for commit